lib/downloader.rb

```diff
@@ -388,12 +389,12 @@ def run_download
     end
     init_raw_dir
     if old_toc.empty? || @force
-      update_subtitles = latest_toc["subtitles"]
+      update_subtitles = latest_toc_subtitles
```

Comment:
__[rubocop]__
* Style/IndentationWidth: Use 2 (not -17) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

<!-- Comment by [SideCI](https://sideci.com) -->